r/.gitignore

```diff
@@ -18,6 +18,7 @@ vignettes/nyc-taxi/
 arrow_*.tar.gz
 arrow_*.tgz
 extra-tests/files
+.deps
```

Comment:
I added this because it got a lot of crap in it while building the site; lmk if there's a good reason to not add it.

Reply:
No objection; what creates this?

Reply:
When I run `pkgdown::build_site()` within RStudio, during the "Installing package into temporary library" it creates this directory.